task_DEV_4/task_DEV_4/EntryPoint.cs

```diff
@@ -5,24 +5,29 @@ namespace task_DEV_4
     class EntryPoint
     {
         static void Main(string[] args)
-        {            
-            Rectangle rectangle = new Rectangle();
-            int height = 5;
-            int width = 6;
-            rectangle.SetRectangleParameters(height, width);            
-            int rectanglePerimeter = rectangle.CalculatePerimeter();
-            Console.WriteLine($"The perimetr of a rectangle is {rectanglePerimeter}");
-            int rectangleArea = rectangle.CalculateArea();
-            Console.WriteLine($"The area of a rectangle is {rectangleArea}");
-            bool isSquare = rectangle.IsSquare();
-            if (rectangle.IsSquare())
+        {
+            try
             {
-                Console.WriteLine("Square");
+                Square square = new Square(3);
+                double squarePerimeter = square.CalculatePerimeter();
+                Console.WriteLine($"The perimetr of a square is {squarePerimeter}");
+                double squareArea = square.CalculateArea();
+                Console.WriteLine($"The area of a sqare is {squareArea}");
+
+                Rectangle rectangle = new Rectangle(5, 6);
+                double rectanglePerimeter = rectangle.CalculatePerimeter();
+                Console.WriteLine($"The perimetr of a rectangle is {rectanglePerimeter}");
+                double rectangleArea = rectangle.CalculateArea();
+                Console.WriteLine($"The area of a rectangle is {rectangleArea}");
+
+                Parallelogram parallelogram = new Parallelogram(3, 6, 40);
+                double parallelogramArea = parallelogram.CalculateArea();
+                Console.WriteLine($"The perimetr of a parallelogram is {parallelogramArea}");
             }
-            else
+            catch (ArgumentOutOfRangeException)
             {
-                Console.WriteLine("Rectangle");
-            }            
+                Console.WriteLine("The parameter cannot be less than or equal to zero.");
```

Comment:
`less or equal to` instead of `less than or equal to`

Reply:
Done